pyproject.toml

```diff
@@ -85,10 +85,17 @@ warn_unreachable = true
 warn_unused_configs = true
 warn_unused_ignores = true
 
+[[tool.mypy.overrides]]
+module = "archinstall.default_profiles.applications.*"
+disallow_any_explicit = true
```

Comment:
@correctmost will this prevent any future use of the `_: Any`? 
We're using that for translations, so it's likely we'll need it again

Reply:
Great work overall with these quality improvements, it's understated how much we appreciated it I think!